gateway/gateway.go

```diff
@@ -745,12 +746,6 @@ func (g *gateway) receivePackets(ctx context.Context) {
 					if isDuplicate {
 						continue
 					}
-					minSNR := sfToSNRMin[packet.DataRate]
-					if float64(packet.SNR) < minSNR {
-						g.logger.Warnf("packet skipped due to low signal noise ratio: %v, min is %v", packet.SNR, minSNR)
```

Comment:
so felix had initially asked us to make this a warning instead of an error. now that we are switching back to an error below we should prob let him know